src/gpt_computer_agent/audio/record.py

```
@@ -2,12 +2,12 @@
     from ..gui.signal import *
     from ..utils.db import *
     from ..utils.telemetry import my_tracer, os_name
-    from ..gpt_computer_agent import the_input_box
+    from .. import the_input_box
 except ImportError:
     from gui.signal import *
```

Ruff (0.13.1)

`from gui.signal import *` used; unable to detect undefined names

(F403)